sdk/python/feast/offline_store.py

```diff
@@ -87,8 +87,14 @@ class OfflineStore(ABC):
     @staticmethod
     @abstractmethod
     def pull_latest_from_table(
-        feature_view: FeatureView, start_date: datetime, end_date: datetime,
-    ) -> Optional[pyarrow.Table]:
+        data_source: DataSource,
+        entity_names: List[str],
+        feature_names: List[str],
+        event_timestamp_column: str,
+        created_timestamp_column: Optional[str],
+        start_date: datetime,
+        end_date: datetime,
+    ) -> pyarrow.Table:
```

Comment:
I think we need comments here to explain the fields. It's a bit confusing because `feature_names` are actually column names as they are found in the remote table, and that isnt immediately obvious.